setup.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+from os import path
+from setuptools import setup, find_packages
+
+this_directory = path.abspath(path.dirname(__file__))
+with open(path.join(this_directory, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+setup(
+    name="golden_hour",
+    version="1.0.0",
+    author="Alan Hussey",
```

Comment:
Is the `author` field required? I think it's inaccurate to put just my name.

Reply:
nope, I'll remove it